src/sentry/api/helpers/group_index/delete.py

```diff
@@ -61,7 +55,7 @@ def delete_group_list(
 
     countdown = 3600
     # With ClickHouse light deletes we want to get rid of the long delay
-    if issue_platform_deletion_allowed and not error_group_found:
```

Comment:
I can't highlight it, but does `call_delete_seer_grouping_records_by_hash` need to be called for non-error issues?

Reply:
Opened this: https://github.com/getsentry/sentry/pull/90742